WebHostLib/misc.py

```diff
@@ -37,17 +37,31 @@ def start_playing():
     return render_template(f"startPlaying.html")
 
 
+# TODO for back compat. remove around 0.4.5
 @app.route('/weighted-settings')
 @cache.cached()
-def weighted_settings():
-    return render_template(f"weighted-settings.html")
+def route_weighted_options():
+    return redirect("/weighted-options", 301)
```

Comment:
I think this is how it's supposed to be done:
```suggestion
    return redirect(url_for('player_options', game=game), 301)
```